### PR TITLE
feat: separate registry check from certificate checks

### PR DESCRIPTION
**Pull request by Nebulis**
## Goal

image

### Business rules
1. For the green/gray header
- if one of the issuer is in the registry => display the green header (in the registry)
- if all issuers are not in the registry => display the gray header (*NOT* in the registry)

2. For the Certificate checks
- if one of the issuer is in the registry display the first available issuer name
- if all issuers are not in the registry, display the first available name from dns resolution. If all dns resolution fails, displays unknown

### Changes from the image
The link in case of not in the registry must be *What does this mean ?*

## Help me
In the code, `CertificateVerifyBlock` and `DetailedCertificateVerifyBlock` checks for the validity of many status, but from my understanding we dont display the certificate in most of the cases (all ?), should I rake the opportunity to remove everything that is unecessary ?

### CODE REVIEW

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.test.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ describe("Certificate verify block getIdentityVerificationText", () => {
         { registry: "demo", dns: "demo.com" }
       ];
       expect(getIdentityVerificationText(testValue)).toEqual(
-        "Accredited by SSG"
+        "Certificate issued by govtech"
```
Comment (sumitnitsurat) [inline src/components/CertificateVerifyBlock/CertificateVerifyBlock.test.js]: I think its better if we only have the capitalized name of the organization.

**Comment by sumitnitsurat** [comment]
Screenshot 2019-10-14 at 10 49 21


"Ropsten" is coming with the name of the institute. Is it intentionally done?

**Comment by Nebulis** [comment]
It's the name as available in registry.json. UI looks weird on your screenshot :see_no_evil:

**Comment by Nebulis** [comment]
My bad it looks ok =)

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.test.js (hunk 1 above)
Comment (Nebulis) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.test.js]: you mean displaying `Govtech` or `GOVTECH` in that case ?

Comment (sumitnitsurat) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.test.js]: yes. either one. In the previous implementation, we went with all capital.

Comment (Nebulis) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.test.js]: I dunno I'm not fond of transforming data they set themself :thinking: I will uppercase to keep the behaviour like before

Comment (Nebulis) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.test.js]: done

File: src/components/CertificateViewer.js [hunk 2]
```diff
@@ -101,15 +103,44 @@ const renderHeaderBlock = props => {
   );
 };
 
-const CertificateViewer = props => {
+export const StatelessCertificateViewer = props => {
```
Comment (sumitnitsurat) [inline src/components/CertificateViewer.js]: I dont think naming them stateless is good idea.

Comment (Nebulis) [reply src/components/CertificateViewer.js]: it is and there is already a Certificate viewer component, I dont want to refactor this now :)

**Pull request merged**